src/solvers/dgmulti/flux_differencing_compressible_euler.jl

```diff
@@ -0,0 +1,102 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+
+# TODO: Upstream, LoopVectorization
+#       At the time of writing, LoopVectorization.jl cannot handle this kind of
+#       loop optimally when passing our custom functions `cons2entropy` and
+#       `entropy2cons`. Thus, we need to insert the physics directly here to
+#       get a significant runtime performance improvement.
+function entropy_projection!(cache, u::StructArray{<:SVector{4, <:Union{Float32, Float64}}},
+                             mesh::VertexMappedMesh, equations::CompressibleEulerEquations2D, dg::DGMulti)
```

Comment:
```suggestion
                             mesh::VertexMappedMesh, equations::CompressibleEulerEquations2D, dg::DGMultiFluxDiff{<:Polynomial})
```

For general "modal" schemes, this would be a great default. However, the Gauss collocation solver I'm working on next might benefit from a slightly different specialization so I think we should restrict this to `approximation_type=Polynomial()` `DGMulti` solvers for now.